tasks/xpack/security/elasticsearch-security-native.yml

```diff
@@ -73,7 +102,7 @@
 #Delete all non required users NOT inc. reserved
 - name: Delete Native Users
   uri:
-    url: http://{{es_api_host}}:{{es_api_port}}/_xpack/security/user/{{item}}
+    url: "https://{{es_api_host}}:{{es_api_port}}/{{es_security_endpoint}}/user/{{item}}"
```

Comment:
I don't think this one can be hardcoded as it will break for anybody using `es_api_scheme: "http"`.